spec/requests/partner_group_spec.rb

```diff
@@ -0,0 +1,27 @@
+RSpec.describe "PartnerGroups", type: :request do
+  let(:user) { create(:user) }
+  let(:partner_group) { create(:partner_group) }
+
+  before do
+    sign_in(user)
+  end
+
+  describe "DELETE #destroy" do
```

Comment:
There should be a spec that checks that the delete button is only shown if there are no partners. This can still be a request spec since you can check the response HTML.

Reply:
Also - please don't force push your branch once a review has started - it becomes harder to see what has changed since the last review.

Reply:
Hii @dorner, 
I've updated the spec based on your suggestion. I had to force-push the changes since I couldn't see my updates in the branch. Please ignore this for now. I'll work on improving it in future PRs